src/scss/components/_resource-detail.scss

```
@@ -0,0 +1,52 @@
+.fill-465d8b {
+	fill: var(--currentColor, #465d8b);
+}
+
+.stroke-465d8b {
+	stroke: var(--currentColor, #465d8b);
+}
+
+.fill-stroke-465d8b {
+	fill: var(--currentColor, #465d8b);
+	stroke: var(--currentColor, #465d8b);
+}
+
+.fill-white-stroke-465d8b {
+	fill: var(--bColor, white);
+	stroke: var(--currentColor, #465d8b);
+}
+
+// Resource detail page
+.resource-detail {
+	h1 {
+		// margin-bottom: 0;
+	}
+
+	.features {
+		display: grid;
+		grid-template-columns: rem(80) auto;
+		row-gap: rem(32);
+
+		svg {
+			height: rem(48);
+			width: rem(48);
+		}
+
+		div {
+			align-self: center;
+		}
+
+		.feature-name {
+			color: $green-dark;
+			font-weight: $font-weight-bold;
+		}
+
+		.learn-tag {
+			background-color: $grey-light;
+			border-radius: rem(22);
+			display: inline-block;
+			margin: rem(2) rem(8);

```

```suggestion
			margin: rem(2) 0 rem(2) rem(8);
```

The spacing between tags is a little large, so applying on the left matches the design more closely